gitops/argocd/tektoncd/tekton-chains.yaml

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
I'm unable to find how this is being used? Perhaps, you need to include this in kustomization.yaml? 

But one step back, why do we need this at all? On a cluster with openshift pipelines operator installed, we would automatically have TektonChain CRD available no?